File: ATV_1_Estruturas_de_selecao/exercicio2.hs
categorias = ["INFANTIL A", "INFANTIL B", "JUVENIL A", "JUVENIL B", "SENIOR"]

verificaCategoria idade
    |idade >= 5 && idade <= 7 = categorias !! 0
    |idade >= 8 && idade <= 10 = categorias !! 1
    |idade >= 11 && idade <= 13 = categorias !! 2
    |idade >= 14 && idade <= 17 = categorias !! 3
    |idade >= 18 = categorias !! 4
    |otherwise = "Categoria nao listada"
    

File: ATV_1_Estruturas_de_selecao/exercicio2.py
categorias = [
    "INFANTIL A",
    "INFANTIL B",
    "JUVENIL A",
    "JUVENIL B",
    "SENIOR"
]

idade = int(input("Digite a idade do competidor: "))

if idade >= 5 and idade <=7:
    print("A categoria do competidor é: {}".format(categorias[0]))
elif idade >= 8 and idade <= 10:
    print("A categoria do competidor é: {}".format(categorias[1]))
elif idade >= 11 and idade <= 13:
    print("A categoria do competidor é: {}".format(categorias[2]))
elif idade >= 14 and idade <= 17:
    print("A categoria do competidor é: {}".format(categorias[3]))
elif idade >= 18:
    print("A categoria do competidor é: {}".format(categorias[4]))
else:
    print("Categoria não listada")
    